server/spec/models/auth_token_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
Can we add this to the `server/.rspec` file so we don't have to require it in every spec?

Afaik, the only reason not to do that is if we're going to have a lot of tests we'll want to run that don't require us to load Rails, but that shouldn't be the case here?

Reply:
Good callout. I'll fix that.